src/pilot/install-director.sh

```diff
@@ -266,6 +266,13 @@ apply_patch "sudo patch -b -s /usr/lib/python2.7/site-packages/dracclient/resour
 sudo rm -f /usr/lib/python2.7/site-packages/dracclient/resources/raid.pyc
 sudo rm -f /usr/lib/python2.7/site-packages/dracclient/resources/raid.pyo
 
+# This hacks in a patch to remove this hack as OOB introspection is fixed. 
```

Comment:
This comment is not clear to me. How about something like the following?

    # This hacks in a patch to out-of-band inspection that sets pxe_enabled on ports.

Reply:
It's updated in latest commit